packages/augur-ui/src/modules/auth/actions/load-account-data.ts

```diff
@@ -34,6 +35,11 @@ export const loadAccountData = (
   dispatch(loadAccountHistory());
   dispatch(checkAccountAllowance());
   dispatch(updateAssets());
+  // Only call loadUniverseDetails (to display Universe Picker in
+  // account dropdown) if a fork has actually occurred
+  if (universe.forkingInfo !== null) {
+    dispatch(loadUniverseDetails(universe.id,loginAccount.address));
```

Comment:
user address is already available. Dont have to use loginAccount.address. Just address